test/test_log.rb

```diff
@@ -404,6 +409,43 @@ def test_level_reload
     # check fluentd log side level is also changed
     assert_equal(Fluent::Log::LEVEL_DEBUG, log.level)
   end
+
+  DAY_SEC = 60 * 60 * 24
+  data(
+    rotate_daily_age: ['daily', 100000, DAY_SEC + 1],
+    rotate_weekly_age: ['weekly', 100000, DAY_SEC * 7 + 1],
+    rotate_monthly_age: ['monthly', 100000, DAY_SEC * 31 + 1],
+    rotate_size: [1, 100, 0, '0'],
+  )
+  def test_log_with_logdevio(expected)
+    old = ENV['TZ']
```

Comment:
Use `with_timezone` test helper method defined in `lib/fluent/test/helpers.rb` instead of doing by itself.